lessons/de/chapter_5.yaml

```diff
@@ -0,0 +1,224 @@
+    überwältigend. Es ist wichtig, sich daran zu erinnern 
+
+    Letztlich dienen die Regeln, die wir aufzeigen, nicht dazu, Ihnen das Leben schwer zu machen, sondern um Ihnen zu helfen
+    Sie machen Ihren Code weniger fehleranfällig!
+- title: Eigentum
+  content_markdown: >
+    Durch die Instanziierung eines Typs und dessen **Bindung** an einen Variablennamen wird ein Speicher erstellt
+    Ressource, die der Rust-Compiler validiert
+    
+    durch sein ganzes **Leben**. Die gebundene Variable wird als Ressource bezeichnet
+    **Eigentümer**.
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=struct%20Foo%20%7B%0A%20%20%20%20x%3A%20i32%2C%0A%7D%0A%0Afn%20main()%20%7B%0A%20%20%20%20%2F%2F%20Instan%C8%9Biem%20structuri%20(Foo)%20%C8%99i%20le%20leg%C4%83m%20de%20variabile%20(foo)%0A%20%20%20%20%2F%2F%20pentru%20a%20crea%20resurse%20%C3%AEn%20memorie%0A%20%20%20%20let%20foo%20%3D%20Foo%20%7B%20x%3A%2042%20%7D%3B%0A%20%20%20%20%2F%2F%20foo%20este%20proprietarul%0A%7D%0A
+- title: Bereichsbasiertes Ressourcenmanagement
+  content_markdown: |
+    Rust verwendet das Ende des Gültigkeitsbereichs als Ort zum Dekonstruieren und Freigeben einer Ressource.
+
+    Der Begriff für diese Dekonstruktion und Freigabe wird **Drop** genannt.
+
+    Speicherdetail:
+    * Rust verfügt über keine Garbage Collection.
+    * Dies wird in C++ auch als Resource Acquisition Is Initialization (RAII) bezeichnet.
+  code: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=struct%20Foo%20%7B%0A%20%20%20%20x%3A%20i32%2C%0A%7D%0A%0Afn%20main()%20%7B%0A%20%20%20%20let%20foo_a%20%3D%20Foo%20%7B%20x%3A%2042%20%7D%3B%0A%20%20%20%20let%20foo_b%20%3D%20Foo%20%7B%20x%3A%2013%20%7D%3B%0A%0A%20%20%20%20println!(%22%7B%7D%22%2C%20foo_a.x)%3B%0A%0A%20%20%20%20println!(%22%7B%7D%22%2C%20foo_b.x)%3B%0A%20%20%20%20%2F%2F%20se%20renun%C8%9B%C4%83%20la%20foo_b%20aici%0A%20%20%20%20%2F%2F%20se%20renun%C8%9B%C4%83%20la%20foo_a%20aici%0A%7D%0A
+- title: Das Löschen ist hierarchisch
+  content_markdown: >
+    Wenn eine Struktur gelöscht wird, wird zuerst die Struktur selbst und dann die Struktur selbst gelöscht
+    Kinder werden einzeln abgesetzt und so weiter.
+
+
+    Speicherdetails:
+
+    * Durch die automatische Freigabe von Speicher trägt Rust dazu bei, dass es weniger Speicher gibt
+    Speicherlecks.
+
+    * Speicherressourcen können nur einmal gelöscht werden.
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=struct%20Bar%20%7B%0A%20%20%20%20x%3A%20i32%2C%0A%7D%0A%0Astruct%20Foo%20%7B%0A%20%20%20%20bar%3A%20Bar%2C%0A%7D%0A%0Afn%20main()%20%7B%0A%20%20%20%20let%20foo%20%3D%20Foo%20%7B%20bar%3A%20Bar%20%7B%20x%3A%2042%20%7D%20%7D%3B%0A%20%20%20%20println!(%22%7B%7D%22%2C%20foo.bar.x)%3B%0A%20%20%20%20%2F%2F%20se%20renun%C8%9B%C4%83%20la%20foo%20mai%20%C3%AEnt%C3%A2i%0A%20%20%20%20%2F%2F%20apoi%20se%20renun%C8%9B%C4%83%20la%20foo.bar%0A%7D%0A
```

Comment:
Please translate the code to German or use the English version instead of the Romanian version.